chain/network/src/store/schema/mod.rs

```diff
@@ -101,6 +101,36 @@ impl BorshRepr for KnownPeerStateRepr {
     }
 }
 
+/// A Borsh representation of the primitives::ConnectionInfo.
+#[derive(BorshSerialize, BorshDeserialize)]
+pub struct ConnectionInfoRepr {
```

Comment:
can it be private? Or at least pub(super)?

Reply:
It can be pub(super). It seems there are many other public structs in this file which could also have their visibility downgraded.